test/helm/default.yaml

```diff
@@ -369,7 +369,7 @@ spec:
         - name: KUBERNETES_CLUSTER_DOMAIN
           value: cluster.local
         image: docker.io/envoyproxy/gateway-dev:latest
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
```

Comment:
curious how it defaults to `IfNotPresent` when the default is `Always` in the Makefile
https://github.com/envoyproxy/gateway/blob/9a626f262317b90946219d7f85069221cce2c4b6/tools/make/kube.mk#L65

Reply:
is this used for helm install/template?